src/Sql/dbo/Stored Procedures/SsoConfig_ReadManyByNotBeforeRevisionDate.sql

```diff
@@ -0,0 +1,14 @@
+CREATE PROCEDURE [dbo].[SsoConfig_ReadManyByNotBeforeRevisionDate]
+    @NotBefore DATETIME2(7)
```

Comment:
Also known as "after"? :-D

Reply:
Sorry, lol, my head was in SSO world, lol